src/DataStructures.jl

```diff
@@ -72,6 +72,145 @@ function Base.get!(
     return get!(default, cache.cache, key)
 end
 
+"""
+    Base.get(cache::LRUCache{K, V}, key::K, default::V)
+
+Get the value associated with `key` in `cache`. If the key is not in the
+cache, return the `default`. If the `key` is in `cache`, update the key's priority.
```

Comment:
```suggestion
Return the value associated with `key` in `cache`. If `key` is not in `cache`,
return `default`. 
```

The existance of priority is an implementation detail that users don't need to know about.